src/NuGet.Core/NuGet.Configuration/Utility/SettingsUtility.cs

```diff
@@ -104,6 +104,23 @@ public static SignatureValidationMode GetSignatureValidationMode(ISettings setti
             return SignatureValidationMode.Accept;
         }
 
+        public static bool GetForceUpdatePackageLastAccessTimeEnabledStatus(ISettings settings)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException(nameof(settings));
+            }
+
+            var forceUpdatePackageLastAccessTimeStatus = GetConfigValue(settings, ConfigurationConstants.ForceUpdatePackageLastAccessTime);
```

Comment:
Does this have to be nuget.config setting? Maybe env var approach could be appropriate.
Please see similar perf effort: https://github.com/NuGet/NuGet.Client/pull/4220

Reply:
Hmm sure, if you prefer env var, no strong feelings.

Reply:
Sorry to contradict @erdembayar, but I don't consider the "enhanced retries" work as a good example because initially it was experimental, and we don't want to go through the effort of making experimental features use nuget.config as it's a lot more work.

Take a look at [our nuget.config docs](https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file), and several nuget.config settings have environment variable overrides, like `globalPackagesFolder` -> `NUGET_PACKAGES`, `http_proxy`, `no_proxy`.

I'd argue that these older configs are precedent, and that enhanced http retries, now that it's no longer experimental, is doing the wrong thing. Personally, I like the idea of having it as a nuget.config setting. I'm neutral on whether or not it should have an environment variable override, but for what it's worth, the global packages override is implemented in `SettingsUtility.cs`, so if we want to make it available as both nuget.config and env var, that's probably where it should be implemented.